cli/cmd/cloud_account.go

```diff
@@ -12,9 +14,75 @@ var (
 		Short:   "Manage cloud accounts",
 		Long:    "Manage cloud account integrations with Lacework",
 	}
+
+	// used by cloud account list to list only a single type of cloud account
+	cloudAccountType string
+
+	// cloudAccountsListCmd represents the list sub-command inside the cloud accounts command
+	cloudAccountListCmd = &cobra.Command{
+		Use:   "list",
+		Short: "List all available cloud account integrations",
+		Args:  cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			var (
+				cloudAccounts api.CloudAccountsResponse
+				err           error
+			)
+
+			if cloudAccountType != "" {
+				caType, found := api.FindCloudAccountType(cloudAccountType)
+				if !found {
+					return errors.Errorf("unknown cloud account type '%s'", cloudAccountType)
+				}
+				cloudAccounts, err = cli.LwApi.V2.CloudAccounts.ListByType(caType)
+			} else {
+				cloudAccounts, err = cli.LwApi.V2.CloudAccounts.List()
+			}
+			if err != nil {
+				return errors.Wrap(err, "unable to get cloud accounts")
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(cloudAccounts.Data)
+			}
+
+			if len(cloudAccounts.Data) == 0 {
+				cli.OutputHuman("There was no cloud account found.\n")
```

Comment:
This command seems to be having a `plural vs. singular` crisis.  I notice some of this was here before you added anything, but:

1. The API is `cloud_accounts`
2. The file is name `cloud_account`.
3. The command is `cloud-account` but aliased to `cloud-accounts`
4. In some message we say `unable to get cloud accounts`, but in other messaging `there was no cloud account`, yet we're listing multiple `accounts`...

I would actually be in favor of this command being `cloud-accounts` primary and aliased to `cloud-account`, and all of the messaging being plural in nature.  For instance `No cloud accounts found.`

Reply:
Actually, looking at the list of commands (policy / query / integration / etc.) it seems like most things are in fact singular.  Perhaps just clean up that one `OutputHuman` to be `No cloud accounts found.`...